tools/esdlcmd/esdl2ecl.cpp

```diff
@@ -389,6 +391,7 @@ class Esdl2EclCmd : public EsdlConvertCmd
                 "      --ecl-imports         Comma-delimited import list to be attached to output ECL\n"
                 "                            each entry generates a corresponding import *.<entry>\n"
                 "      --ecl-header          Text included in target header (must be valid ECL) \n"
+                "      --utf8-               Don't use UTF8 strings"
```

Comment:
I assume the "-" postfix is the notation to negate the option?

Reply:
Yes.  In reality I think it should always be UTF8 so this notation is only meant if for some reason someone wants to turn it off.  I hope no one needs to.. but maybe during the transition.

Reply:
By the way the postfix is just the standard used in Gavins parameter processor.  The option works without, or with plus, but both of those are the default, so I just show the one way that actually changes the default.